Docs/Book/GettingStartedInPython.rst

```diff
@@ -1417,8 +1417,38 @@ or into a generic framework:
 Maximum Common Substructure
 ***************************
 
-The FindMCS function find a maximum common substructure (MCS) of two
-or more molecules:
+There are 2 methods for finding maximum common substructures.  The first, FindMCS,
+finds a single fragment maximum common substructure (MCS) of two or more molecules:
+The second, RascalMCES, finds the maximum common edge substructure (MCES) between two
+molecules and can return a multi-fragment MCES.  The difference is demonstrated with the
+following pair of molecules:
+
++-------------------------------------+
+| .. image:: images/mcs_example_1.png |
++-------------------------------------+
+| .. image:: images/mcs_example_2.png |
++-------------------------------------+
```

Comment:
I think you forgot to add the images to git.

Reply:
Doh!  Thanks.